master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -249,9 +249,11 @@ private[celeborn] class Master(
       val pushPort = pbRegisterWorker.getPushPort
       val fetchPort = pbRegisterWorker.getFetchPort
       val replicatePort = pbRegisterWorker.getReplicatePort
-      val disks = pbRegisterWorker.getDisksList.asScala
-        .map { pbDiskInfo => pbDiskInfo.getMountPoint -> PbSerDeUtils.fromPbDiskInfo(pbDiskInfo) }
-        .toMap.asJava
+      val disks = new util.HashMap[String, DiskInfo]()
```

Comment:
this should not change then

Reply:
Reverted, though I prefer to change because it avoids some conversions between java and scala. But it's trivial, both are fine to me :)